app/models/post.rb

```diff
@@ -0,0 +1,16 @@
+class Post < ApplicationRecord
+  belongs_to :user
+  has_many :likes, :comments
+
+  after_save :update_user_post_counter
+
+  def return_most_recent_comments
+    comments.last(5)
```

Comment:
- [ ] <b>REQUIRED:</b> Here too. Kindly get the last comments by date. 

```ruby
comments.order('created_at Desc').limit(5)

Reply:
This change has been done well